detour_test.go

```diff
@@ -147,6 +147,8 @@ func TestClosing(t *testing.T) {
 }
 
 func TestIranRules(t *testing.T) {
+	t.Helper()
```

Comment:
Why adding this? I can not see any difference on the console output after removing this.

Reply:
Oh oops, I had temporarily created a helper method and this was left over from it. Will remove.